chainer/iterators/multiprocess_iterator.py

```diff
@@ -62,13 +62,17 @@ def __init__(self, dataset, batch_size, repeat=True, shuffle=True,
 
         self._finalized = None
 
+        self._previous_epoch_detail = -1.
```

Comment:
There is another solution. We initialize `previous_epoch_detail` with `None` and convert it to `-1` in `serialize` method. In this case, we can remove getter method.